File: Source/Rio.Models/DataTransferObjects/WaterTransfer/WaterTransferDto.cs
using System;

namespace Rio.Models.DataTransferObjects.WaterTransfer
{
    public class WaterTransferDto
    {
        public int WaterTransferID { get; set; }
        public int OfferID { get; set; }
        public DateTime TransferDate { get; set; }
        public int TransferYear { get; set; }
        public int AcreFeetTransferred { get; set; }
        public decimal? UnitPrice { get; set; }
        public string Notes { get; set; }
        public string TradeNumber { get; set; }

        public WaterTransferRegistrationSimpleDto BuyerRegistration { get; set; }
        public WaterTransferRegistrationSimpleDto SellerRegistration { get; set; }
    }
}

File: Source/Rio.Web/src/app/shared/models/water-transfer-dto.ts
import { WaterTransferRegistrationSimpleDto } from './water-transfer-registration-simple-dto';

export class WaterTransferDto {
    WaterTransferID: number;
    OfferID: number;
    TransferDate: Date;
    TransferYear: number;
    AcreFeetTransferred: number;
    UnitPrice: number;
    BuyerRegistration: WaterTransferRegistrationSimpleDto;
    SellerRegistration: WaterTransferRegistrationSimpleDto;
    Notes: string;
    TradeNumber: string;

    constructor(obj?: any) {
        Object.assign(this, obj);
    }
}

